### PR TITLE
Add charles-nurse.md to speaker directory

### PR DESCRIPTION
**Pull request by cnurse**
Submitting my application to be included in speaker directory

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=783) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=783) before we can accept your contribution.<br/><br/>:x: cnurse [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=783)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=783) it.</sub>

**Review by shawnwildermuth — CHANGES_REQUESTED** [review]
Please change to use https on all links.

**Review by Arhell — APPROVED** [review]

**Comment by shawnwildermuth** [comment]
Any word on making these changes?

File: input/community/speakers/charles-nurse.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - Blazor
+  - Web Development
+Blog: http://www.charlesnurse.com
+Twitter: https://twitter.com/cnurse
+LinkedIn: https://www.linkedin.com/in/charlesnurse/
+Sessionize: https://sessionize.com/charles-nurse
+---
+Charles Nurse is a Pluralsight Author, an 11-time Microsoft MVP (2007-2018) and an ASPInsider (since 2008).   He is a successful .NET Architect and a Graph enthusiast, and he has led development teams on both Open Source projects and large Enterprise projects.
+
+He has spoken at many major conferences including the Software Developers Conference (SDC) (www.sdc.nl),   DevConnections (www.devconnections.com),  DevTeach (www.devteach.com), VSLive (www.vslive.com), Prairie DevCon(www.prairiedevcon.com) and DevReach(www.DevReach.com).    He is a frequent speaker at local User Groups and Code Camps, including .NET BC, VBug, Vancouver TechFest and Microsoft Developer days.
```
Comment (SeanKilleen) [inline input/community/speakers/charles-nurse.md:21]:
```suggestion
He has spoken at many major conferences including the Software Developers Conference (SDC) (https://www.sdc.nl),   DevConnections (https://www.devconnections.com),  DevTeach (https://www.devteach.com), VSLive (https://www.vslive.com), Prairie DevCon(https://www.prairiedevcon.com) and DevReach (https://www.DevReach.com).    He is a frequent speaker at local User Groups and Code Camps, including .NET BC, VBug, Vancouver TechFest and Microsoft Developer days.
```

Adding https links as requested. @cnurse, if you accept these suggestions, that should validate this PR I think and allow it to be merged.

**Pull request merged**